### PR TITLE
Support for docker image tags

### PR DESCRIPTION
**Pull request by font**
Signed-off-by: Ivan Font <ivan.font@redhat.com>

Fixing issues arising from using a different docker image tag from the default 'latest'.

### CODE REVIEW

**Comment by leseb** [comment]
@font looks like #873 is addressing the same problem already.

File: roles/ceph-mds/tasks/docker/main.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@
 - include: pre_requisite.yml
 - include: "{{ playbook_dir }}/roles/ceph-common/tasks/docker/fetch_image.yml"
   vars:
-    ceph_docker_username: '{{ ceph_mds_docker_username}}'
-    ceph_docker_imagename: '{{ ceph_mds_docker_imagename}}'
+    ceph_docker_username: '{{ ceph_mds_docker_username }}'
```
Comment (leseb) [inline roles/ceph-mds/tasks/docker/main.yml]: I think we should use double quotes here.

File: roles/ceph-mon/tasks/docker/main.yml [hunk 2]
```diff
@@ -24,6 +24,7 @@
   vars:
     ceph_docker_username: '{{ ceph_mon_docker_username}}'
```
Comment (leseb) [inline roles/ceph-mon/tasks/docker/main.yml]: ditto + `ceph_mon_docker_username }}`

File: roles/ceph-nfs/tasks/docker/main.yml [hunk 3]
```diff
@@ -22,8 +22,9 @@
 
 - include: "{{ playbook_dir }}/roles/ceph-common/tasks/docker/fetch_image.yml"
   vars:
-    ceph_docker_username: '{{ ceph_nfs_docker_username}}'
-    ceph_docker_imagename: '{{ ceph_nfs_docker_imagename}}'
+    ceph_docker_username: '{{ ceph_nfs_docker_username }}'
```
Comment (leseb) [inline roles/ceph-nfs/tasks/docker/main.yml]: ditto

File: roles/ceph-restapi/tasks/docker/main.yml [hunk 4]
```diff
@@ -2,8 +2,9 @@
 - include: pre_requisite.yml
 - include: "{{ playbook_dir }}/roles/ceph-common/tasks/docker/fetch_image.yml"
   vars:
-    ceph_docker_username: '{{ ceph_restapi_docker_username}}'
-    ceph_docker_imagename: '{{ ceph_restapi_docker_imagename}}'
+    ceph_docker_username: '{{ ceph_restapi_docker_username }}'
```
Comment (leseb) [inline roles/ceph-restapi/tasks/docker/main.yml]: ditto

File: roles/ceph-rgw/tasks/docker/main.yml [hunk 5]
```diff
@@ -19,8 +19,9 @@
 - include: pre_requisite.yml
 - include: "{{ playbook_dir }}/roles/ceph-common/tasks/docker/fetch_image.yml"
   vars:
-    ceph_docker_username: '{{ ceph_rgw_docker_username}}'
-    ceph_docker_imagename: '{{ ceph_rgw_docker_imagename}}'
+    ceph_docker_username: '{{ ceph_rgw_docker_username }}'
```
Comment (leseb) [inline roles/ceph-rgw/tasks/docker/main.yml]: ditto

**Comment by leseb** [comment]
sorry, would you mind rebasing this? thanks!

**Comment by font** [comment]
Sure - I meant to do that anyway!

**Pull request merged**